### ISSUE TITLE
перевод без осетинского текста (по признанию самого Абаева))))))))

### ISSUE THREAD

**Issue by jvsntsn** [abaevdict/abaevdict-tei#4]
![image](https://user-images.githubusercontent.com/48504970/153709685-aad26eca-2c1c-46e1-8726-4f486e03eea3.png)

**Comment by obelyaev** [comment]
Я думаю, что правильнее всего рассматривать это как пример на русском языке (т.е. у него будет атрибут xml:lang=ru). И в этом случае курсив будет убираться. Надо понять, легко ли это реализовать.

**Comment by obelyaev** [comment]
Собственно, это уже сделано правильно (не знаю, я это сделал или Вы сами :) )

**Comment by jvsntsn** [comment]
![image](https://user-images.githubusercontent.com/48504970/160673947-d194a321-c254-4eda-a343-490712335c78.png)
а вот тут считать примером на русском языке (и самому проставить кавычки)? или как перевод... без осетинского текста? (потому что есть формальный признак - кавычки).

Мне больше нравится первый вариант, раз мы такое допустили. Но на всякий случай уточню.

Слово qan

**Comment by solanum-tuberosum** [comment]
У меня такой же случай. Что делать? Я пока поставила обычные примеры, но не писала ничего в quote. Хотя возможно легче всего было бы разрешить удалять quote и оставлять только русский текст
![image](https://user-images.githubusercontent.com/13102259/163668302-c9cc6cf3-cbd3-4ba4-85c5-10f46f42c224.png)
![image](https://user-images.githubusercontent.com/13102259/163668320-200f6316-5ef5-466e-a436-ab766637a656.png)

**Comment by obelyaev** [comment]
Да, мне кажется, что первый вариант лучше, потому что по сути, я думаю, это тот же случай, что и другие примеры на русском языке. А кавычки можно сделать элементом q и признаком #rend_doubleQuotes при необходимости

**Comment by solanum-tuberosum** [comment]
![image](https://user-images.githubusercontent.com/13102259/169102943-b6f584dc-ed9e-4dd6-ae0e-ff431bff6034.png)
еще встретилась такая штука.

но нельзя будет легко вытащить все примеры, где нет quote? делать руками русский язык и добавлять кавычки это лишняя работа...
